docs/hadoop.adoc

```diff
@@ -121,6 +167,187 @@ def defineGratefulDeadSchema(janusGraph) {
 }
 ----
 
+====== 2. Bulk loading into backend using Spark cluster in standalone mode.
+The steps followed above, under section *Bulk loading into backend using Local Spark* can be followed again to Bulk load data into Janusgraph using Spark.
+
+We will only need to modify our hadoop-bulk-load.properties to point to our Clustered spark, and not local.
+
+Also, in addition to that, we will need to distribute all the necessary Jars (In this case which comes pre-packaged with JanusGraph), so that required classes are loaded in required order and hence doesn't raise conflicts.
```

Comment:
Also, in addition to that, we will need to distribute all the necessary jars which come pre-packaged with JanusGraph,

Reply:
Will do this one. Yours is grammatically more appropriate.